make/common/NativeCompilation.gmk

```diff
@@ -440,76 +440,91 @@ define SetupNativeCompilation
   ifeq ($$($1_STATIC_LIBRARY),)
     ifneq ($$($1_DEBUG_SYMBOLS),)
       ifeq ($(ENABLE_DEBUG_SYMBOLS), true)
-        ifneq ($(OPENJDK_TARGET_OS), macosx) # no MacOS X support yet
-          ifneq ($$($1_OUTPUT_DIR),$$($1_OBJECT_DIR))
+        ifneq ($$($1_OUTPUT_DIR),$$($1_OBJECT_DIR))
+          ifeq ($(OPENJDK_TARGET_OS), macosx)
+	    # MacOS debug symbols are in %.dSYM directories
+	    $$($1_OUTPUT_DIR)/$$($1_BASENAME).dSYM/Contents/Info.plist : $$($1_OBJECT_DIR)/$$($1_BASENAME).dSYM/Contents/Info.plist
```

Comment:
Does this need indenting?

Reply:
no, the make target:dependency rule line is space indented, it is just the recipe lines that must be tabs

Reply:
ah there is a tab on that line, will remove